addons/mail/static/src/js/composers/basic_composer.js

```diff
@@ -658,6 +770,24 @@ var BasicComposer = Widget.extend({
                 this._mentionManager.detectDelimiter();
         }
     },
+    /**
+     * _onPasteImage event is triggered when image type attachment is pasted
+     * from any webpage and it is uploaded as an attachment
+     *
+     * @private
+     * @param {ClipboardEvent} ev
+     */
+    _onPasteImage: function (ev) {
```

Comment:
remove copy-paste feature. It is not working, and after discussing with EST we should drop this feature from this task